sormas-e2e-tests/src/test/resources/features/sanity/web/Contacts.feature

```diff
@@ -8,4 +8,15 @@ Feature: Create contacts
       And I create a new contact
       Then I check the created data is correctly displayed on Edit Contact page
       Then I open Contact Person tab
-      And I check the created data is correctly displayed on Edit Contact Person page
\ No newline at end of file
+      And I check the created data is correctly displayed on Edit Contact Person page
+
+  Scenario: Delete created contact
+    Given I log in with the user
+    And I click on the Contacts button from navbar
+    And I click on the NEW CONTACT button
+    And I create a new contact
+    Then I check the created data is correctly displayed on Edit Contact page
```

Comment:
We are not checking the same actions several times, we already checking this in other test.

Reply:
The UUID and data is collected in this check. I can refactor and move the UUI scrapping in another method/step